docs/Error-Handling.md

```diff
@@ -0,0 +1,9 @@
+<h1 align="center">Fastify</h1>
+
+## Error Handling
+
+Uncaught errors are likely to cause memory leaks, file descriptor leaks and other major production issues ([see this])(http://npm.im/make-promises-safe). [Domains](https://nodejs.org/en/docs/guides/domain-postmortem/) were introduced to try fixing this issue, but they did not. The best way to deal with unhandled errors at the moment is to [crash](https://nodejs.org/dist/latest-v8.x/docs/api/deprecations.html#deprecations_dep0018_unhandled_promise_rejections), given the fact that it is not possible to process all uncaught errors in a sensible way
```

Comment:
This is the correct link for the "crash" (https://nodejs.org/api/process.html#process_warning_using_uncaughtexception_correctly). The same is not true for promises, so you'll need to link that deprecation. I would also link to https://github.com/mcollina/make-promises-safe.